### PR TITLE
Create IMAGE CLASSIFICATION - MACHINE LEARNING

### PR DESCRIPTION
**Pull request by naseeb0**
Object/Poses Classification of User's Choice using JS and Teachable Machine

Thanks for making a PR! Please fill in the below:

Name: Naseeb Dangi

Do you run a club? 
I was leader way before but I think i am not now but I am active member.

What relevant experiences do you have to help you write this Jam?
I have some experiences of Machine Learning and lots of MLH Hackathons.

Will you be available on Slack to provide support to Hack Clubbers who run into issues while going through your Jam?
Yes I would be there to support who run into issue while going through the jam

### CODE REVIEW

File: jams/IMAGE CLASSIFICATION - MACHINE LEARNING [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+_Requirements:_ Computer With Internet Access and Web Browser
+
+_Languages used:_ JavaScript
+
+_Customization opportunity:_ While we'll explore Face Mask Detection, they have the freedom to train your model to recognize any objects you're interested in. 
+Personalize the user interface and add additional features to enhance the classification experience.
+
+Platform Limitations: Browser-based Development, so available on all platforms
+
+_Time estimation_: 1 Hour
```
Comment (miguelaenlle) [inline jams/IMAGE CLASSIFICATION - MACHINE LEARNING:15]: Add time estimates for each step. ex) Setup (5 minutes)

Comment (miguelaenlle) [inline jams/IMAGE CLASSIFICATION - MACHINE LEARNING:15]: Add time estimates for each step, ex. "Setup (5 minutes): ..."

File: jams/IMAGE CLASSIFICATION - MACHINE LEARNING [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### Breakdown
+Here's the breakdown of our exciting journey:
+- Set up: Access the user-friendly Teachable Machine website and create your own image classification project.
+- Part 1: Collect and Prepare Data: Capture images of face masks and examples without face masks to train your model effectively.
+- Part 2: Train Your Model: Utilize the powerful Teachable Machine platform to train your AI model effortlessly.
+- Part 3: Integrate Into Your Web Page: Export your trained model into p5js editor online
```
Comment (miguelaenlle) [inline jams/IMAGE CLASSIFICATION - MACHINE LEARNING]: Provide more instructions for the workshop host on how to incorporate the trained model into a website.

File: jams/IMAGE CLASSIFICATION - MACHINE LEARNING [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Image Classifaction with Teachable Machine - Train and Classify Objects Around You
+### Details:
+_Outcome:_ Create your very own image classification model that runs directly in your web browser and can classify objects/poses of your choice. 
+We are taking Face Mask Detection as Example!
+
+_Requirements:_ Computer With Internet Access and Web Browser
+
+_Languages used:_ JavaScript
```
Comment (miguelaenlle) [inline jams/IMAGE CLASSIFICATION - MACHINE LEARNING]: Mention that it uses HTML and CSS too for web integration

File: jams/IMAGE CLASSIFICATION - MACHINE LEARNING [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+_Time estimation_: 1 Hour
+
+### Breakdown
+Here's the breakdown of our exciting journey:
+- Set up: Access the user-friendly Teachable Machine website and create your own image classification project.
+- Part 1: Collect and Prepare Data: Capture images of face masks and examples without face masks to train your model effectively.
+- Part 2: Train Your Model: Utilize the powerful Teachable Machine platform to train your AI model effortlessly.
+- Part 3: Integrate Into Your Web Page: Export your trained model into p5js editor online
+- Congratulations! You now possess a remarkable image classification model running smoothly in your browser, tailored to your specific object of interest.
```
Comment (miguelaenlle) [inline jams/IMAGE CLASSIFICATION - MACHINE LEARNING:24]: Would add a segment at the end encouraging club members to train their own models

**Review by miguelaenlle — CHANGES_REQUESTED** [review]
Provide more detailed instructions for workshop hosts and time estimates for each segment of the workshop

**Comment by naseeb0** [comment]
I have made the required changes. Can you please look into it.

**Review by miguelaenlle — APPROVED** [review]

**Comment by miguelaenlle** [comment]
I checked out the changes, and they look good. Thank you Naseeb!

**Comment by SerenityUX** [comment]
Hey naseeb0, I love this! We're already using Teachable Machine for a couple workshops. How would you feel about making this with TensorFlow & making it an intermediate/expert difficulty (one step up from just a Teachable Machine Model).

Maybe even incorporate some web scraping? 

https://www.tensorflow.org
https://scrapy.org
https://pypi.org/project/beautifulsoup4/

**Comment by naseeb0** [comment]
I think thats great! but I just wanted to create something that can be accomplished in one hour and give jammers a boost to start learning ML without installing anything anything. 
Maybe we can add addition to this learning or make Part 2 of the Workshop later on?

**Comment by Arpan-206** [comment]
Problem is, it will be counterproductive to have 2 Jams on Teachable Machine.

**Comment by naseeb0** [comment]
So then I will try to submit new workshop asap

**Comment by Arpan-206** [comment]
Sure, looking forward to it!

**Pull request closed without merging**